### PR TITLE
Improve battle booster

### PR DESCRIPTION
**Pull request by sonohoshi**
### Description

1. Add one line popup in booster button
2. Move result VFX to UI from Game scene

### How to test

1. Try to run the booster on a stage that is not completely cleared.

### Screenshot

image

### CODE REVIEW

**Review by Namyujeong — APPROVED** [review]

**Review by hskim881028 — APPROVED** [review]

File: nekoyume/Assets/_Scripts/UI/BattleResult.cs [hunk 1]
```diff
@@ -301,10 +302,10 @@ private void UpdateView(bool isBoosted)
         private IEnumerator CoUpdateViewAsVictory(bool isBoosted)
         {
             AudioController.instance.PlayMusic(AudioController.MusicCode.Win, 0.3f);
-            StartCoroutine(EmitBattleWinVFX(isBoosted));
+            StartCoroutine(EmitBattleWinVFX());
 
             victoryImageContainer.SetActive(true);
-            _victoryImageAnimator.SetInteger("ClearedWave",
+            _victoryImageAnimator.SetInteger(ClearedWave,
                 isBoosted ? 4 : SharedModel.ClearedWaveNumber);
```
Comment (Unengine) [inline nekoyume/Assets/_Scripts/UI/BattleResult.cs:309]: How about make this constant into `const` variable commenting what this line means?

Comment (Unengine) [inline nekoyume/Assets/_Scripts/UI/BattleResult.cs:309]: How about making this constant into const variable commenting what this line means?

Comment (Unengine) [inline nekoyume/Assets/_Scripts/UI/BattleResult.cs:309]: How about making this constant into const variable and commenting what this line means?

**Review by Unengine — APPROVED** [review]

File: nekoyume/Assets/_Scripts/UI/BattleResult.cs (hunk 1 above)
Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/UI/BattleResult.cs:309]: I checked it. I will add comment.

**Review by Unengine — APPROVED** [review]

**Pull request merged**